docs/shared/infra/mav2-client.mdx

```diff
@@ -0,0 +1,30 @@
+<CodeBlocks>
+```ts twoslash title="config.ts"
+import { alchemy, sepolia } from "@account-kit/infra";
+
+const YOUR_API_KEY = "<YOUR_API_KEY>";
+
+export const chain = sepolia;
+
+export const policyId = "<POLICY_ID>";
+
+export const transport = alchemy({
+  apiKey: YOUR_API_KEY,
+});
+```
+
+```ts twoslash title="client.ts"
+import { createModularAccountV2Client } from "@account-kit/smart-contracts";
+import { chain, transport, policyId } from "./config";
+import { LocalAccountSigner } from "@aa-sdk/core";
+import { generatePrivateKey } from "viem/accounts";
+
+export const client = await createModularAccountV2Client({
+  chain,
+  transport,
+  signer: LocalAccountSigner.privateKeyToAccountSigner(generatePrivateKey()),
+  policyId,
+});
```

Comment:
it's generally best to avoid using top-level exported `await`s. any code that imports this module will need to wait for the promise to resolve. also it's not supported by certain environments and bundlers.

some alternatives would be to export a factory function, export a promise (which you can `await`) wherever you use it, or create a singleton and export a function that lazily inits it the first time it's called.